src/+replab/CharacterTable.m

```diff
@@ -0,0 +1,112 @@
+classdef CharacterTable < replab.Obj
+% Describes the character table of a group
+
+    properties (SetAccess = protected)
+        group % replab.Group: group represented by character table
+        irreps % cell of replab.Rep: irreducible representation
+        classes % cell of replab.ConjugacyClass: conjugacy classes
+        chars % matrix(nirreps,nclasses): matrix of characters
+        table % replab.str.Table: table object for display purposes
+    end
+    
+    methods
+
+        function self = CharacterTable(group)
+            self.group = group;
+            % if we have Sn, use Andrew's function
+%             if isa(group, 'replab.SymmetricGroup')
+%                 [classes, irreps, chars] = ...
+%             end
+            [classes, irreps, chars] = self.generateTable;
```

Comment:
Make the constructor of `CharacterTable` plain; and move the computation outside.

So `generateTable` could be a static method `forGenericGroup`.